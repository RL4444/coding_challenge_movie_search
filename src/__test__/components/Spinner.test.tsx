import React from 'react';
import { shallow } from 'enzyme';

import Spinner from '../../components/Spinner';

it('renders the component', () => {
    const component = shallow(<Spinner />);
    expect(component).toMatchSnapshot();
});
