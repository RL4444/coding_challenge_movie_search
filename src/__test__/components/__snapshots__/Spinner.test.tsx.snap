// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<styled.div
  className="spinner "
  height={50}
  style={
    Object {
      "height": 50,
      "width": 50,
    }
  }
  width={50}
>
  <div
    className="overlay "
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
  />
  <div
    className="spinner"
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
  />
</styled.div>
`;
